src/templates/invoice/Invoice.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { FunctionComponent } from "react";
+import { Invoice } from "./types";
+import { format } from "date-fns";
+import styled from "@emotion/styled";
+
+const Container = styled.div`
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  position: relative;
+
+  h1 {
+    color: #4172af;
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  h2 {
+    color: #4172af;
+    font-size: 28px;
+    font-weight: 700;
+  }
+
+  h4 {
+    font-weight: 700;
+  }
+
+  .bg-blue {
+    background-color: #4172af;
+    color: #fff;
+  }
+
+  th {
+    background-color: #4172af;
+    color: white;
+  }
+
+  /*
+
+
+  .table-right {
+    width: 40%;
+    right: 0;
+    top: 100px;
+    position: absolute;
+    text-align: center;
+  }
+
+  .table-left {
+    width: 40%;
+    left: 0;
+    top: 120px;
+    position: relative;
+  }
+
+  .table-full {
+    position: relative;
+    top: 150px;
+  }
+
+  .border {
+    border-style: none none solid solid;
+    border-color: black;
+    border-width: 2px;
+  }
+
+  .total {
+    font-weight: 700;
+    text-align: left;
+    width: 40%;
+    line-height: 10px;
+  }
+
+  .amount {
+    width: 10%;
+    margin-left: 700px;
+  } */
+`;
+
+export const InvoiceTemplate: FunctionComponent<TemplateProps<Invoice>> = ({ document }) => {
+  const { id, date, customerId, terms, billFrom, billTo, billableItems, subtotal, tax, taxTotal, total } = document;
+
+  return (
+    <Container className="p-4 container">
+      <div className="flex flex-wrap">
+        <div className="w-full md:w-5/12 md:ml-auto md:order-2">
+          <h1 className="text-right">INVOICE</h1>
+          <div className="flex flex-wrap bg-blue text-center">
+            <div className="w-1/2 p-1">
+              <p>INVOICE #</p>
+            </div>
+            <div className="w-1/2 p-1">
+              <p>DATE</p>
+            </div>
+          </div>
+          <div className="flex flex-wrap text-center">
+            <div className="w-1/2 p-1">
+              <p>{id}</p>
+            </div>
+            <div className="w-1/2 p-1">
+              <p>{date && format(new Date(date), "d MMM yyyy")}</p>
+            </div>
+          </div>
+          <div className="flex flex-wrap bg-blue text-center">
+            <div className="w-1/2 p-1">
+              <p>CUSTOMER ID</p>
+            </div>
+            <div className="w-1/2 p-1">
+              <p>TERMS</p>
+            </div>
+          </div>
+          <div className="flex flex-wrap text-center">
+            <div className="w-1/2 p-1">
+              <p>{customerId}</p>
+            </div>
+            <div className="w-1/2 p-1">{terms}</div>
+          </div>
+        </div>
+        <div className="w-full md:w-5/12 md:order-1 mb-4 md:mb-0">
+          <h2>{billFrom?.name}</h2>
+          <h3>{billFrom?.streetAddress}</h3>
+          <h3>
+            {billFrom?.city}
+            {`, ${billFrom?.postalCode}`}
```

Comment:
i think the comma will still render regardless of field exist or no, what i can think of is:
```
{billFrom?.postalCode && `, ${billFrom?.postalCode}`}
```